openassessment/xblock/test/test_submission.py

```diff
@@ -132,7 +133,10 @@ def test_prompt_line_breaks(self, xblock):
         # Verify that prompts with multiple lines retain line breaks
         # (backward compatibility in case if prompt_type == 'text')
         resp = self.request(xblock, 'render_submission', json.dumps(dict()))
-        expected_prompt = u"<p><br />Line 1</p><p>Line 2</p><p>Line 3<br /></p>"
+        if django.VERSION[0] >= 2 and django.VERSION[1] > 0:
```

Comment:
I suspect this logic will break for Django 3.0.  You should be able to use `django.VERSION >= (2, 1)` instead.